src/schema/schema.graphql

```diff
@@ -227,6 +227,43 @@ enum LicenseStatus {
 	ACTIVATION_GRACE_PERIOD
 }
 
+enum SearchConversationTaskStatus {
```

Comment:
this is used in tasks as well, we should generically name it as TaskStatus instead of SearchConversationTaskStatus